### PR TITLE
v2: removing v2 http filter protos

### PR DESCRIPTION
**Pull request by alyssawilk**
Risk Level: Low
Testing: n/a
Docs Changes: n/a
Release Notes: n/a

### CODE REVIEW

File: tools/proto_format/proto_sync.py [hunk 1]
```diff
@@ -395,7 +400,10 @@ def generate_current_api_dir(api_dir, dst_dir):
     shutil.rmtree(str(dst.joinpath("service", "auth", "v2alpha")))
 
     for proto in IGNORED_V2_PROTOS:
-        shutil.rmtree(str(dst.joinpath(proto[6:])))
+        try:
+            shutil.rmtree(str(dst.joinpath(proto[6:])))
+        except:
```
Comment (phlax) [inline tools/proto_format/proto_sync.py:405]: debug

**Review by phlax — CHANGES_REQUESTED** [review]
apart from small nit, lgtm - great!

lets hope ci agrees

**Review by htuch — DISMISSED** [review]
LGTM modulo CI and @phlax comment. @tyxia FYI.

**Comment by alyssawilk** [comment]
ok, ci sorted, comment addressed, PTAL

**Review by phlax — APPROVED** [review]
lgtm, thanks @alyssawilk 

it will be good when we dont need the `proto_sync` workaround, its defo confusing - not sure if removing the rest of the proto deps will fix the problem as i think its removing them that makes the diff wrong

anyhow, great, lgtm

**Comment by phlax** [comment]
cc @htuch ^^

**Comment by alyssawilk** [comment]
+mark for shephard LGTM

**Review by htuch — APPROVED** [review]
/lgtm api

**Pull request merged**